### PR TITLE
nix run .#start server 15 local dev full config

### PR DESCRIPTION
**Pull request by samrose**
## What kind of change does this PR introduce?

This PR fixes our `nix run .#start-server 15` function to use configurations found in our ansible files as a basis for configuring postgresql to run locally

Once you run `nix run .#start-server 15` you can then run `nix run .#start-client-and-migrate 15` and you should have a working environment that matches very closely with our AMI release configuration (however it is using the versions in your current branch of your current repo for the .#psql_15/bin bundle).

The logging config in our AMI is commented out so that logs will stream to stdout/stderr for local development

This has been tested on aarch64-linux and aarch64-darwin + x86_64-linux

### CODE REVIEW

File: logfile
Comment (olirice) [inline logfile:1]: Does this logfile need to be committed?

**Review by olirice — APPROVED** [review]

File: logfile
Comment (pcnc) [inline logfile:1]: Approved - also curious about this logfile 👀

**Review by pcnc — APPROVED** [review]

File: logfile
Comment (samrose) [reply logfile:1]: thank you removed

Comment (samrose) [reply logfile:1]: Thanks I removed that

**Pull request merged**